DemoApp-Java/app/build.gradle

```diff
@@ -17,6 +17,10 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
+    compileOptions {
```

Comment:
Do we need to explicitly set this?

Reply:
To use the lambdas I think we do, we don't have it in our other demo app because we haven't switched over to lambdas but we do have it in our test app